frontend/src/pages/add-item/index.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+import FormInput from "../../components/input-text-field";
+import Button from "../../components/button";
+import InputDate from "../../components/inputdate";
+import DropDown from "../../components/dropdown/index.js";
+import "./index.css";
+
+const usersOptions = [
+  {
+    id: 1,
+    user: "user-1",
+  },
+  {
+    id: 2,
+    user: "user-2",
+  },
+];
+
+const AddItemPage = () => {
+  const [itemName, setItemName] = useState("");
+  const [room, setRoom] = useState("");
+  const [reserve, setReserve] = useState("");
+  const [fromDateInput, setFromDateInput] = useState("");
+  const [toDateInput, setToDateInput] = useState("");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(`
+    Item Name: ${itemName}
+    Room: ${room}
+    Reserved User: ${reserve}
+    From: ${fromDateInput}
+    To: ${toDateInput}
+    `);
+  };
+
+  return (
+    <div>
+      <h1>Add Item</h1>
+      <div className="addItemPage">
+        <form onSubmit={handleSubmit}>
+          <FormInput
+            name="itemName"
+            onChange={(e) => setItemName(e.target.value)}
+            placeholder="Item Name"
+            text="Item Name"
+          />
+
+          <FormInput
+            onChange={(e) => setRoom(e.target.value)}
+            placeholder="Room"
+            text="Room"
+          />
+
+          <DropDown
+            label="Reserve Item For"
+            usersOptions={usersOptions}
+            selected={reserve}
+            getReserveData={(reservedata) => setReserve(reservedata)}
```

Comment:
is it possible to use camel casing here?
```suggestion
            getReserveData={(reserveData) => setReserve(reservedata)}
```